src/executables/csql_result.c

```diff
@@ -559,7 +561,8 @@ get_current_result (int **lengths, const CUR_RESULT_INFO * result_info, const CS
       assert (value_type == DB_TYPE_NULL
 	      /* UNKNOWN, maybe host variable */
 	      || result_info->attr_types[i] == DB_TYPE_NULL || result_info->attr_types[i] == DB_TYPE_VARIABLE
-	      || value_type == result_info->attr_types[i]);
+	      || value_type == result_info->attr_types[i]
+	      || (IS_STRING_TYPE (value_type) && IS_STRING_TYPE (result_info->attr_types[i])));
```

Comment:
Does this assertion fail without addition of this line?

Reply:
By changing the return from null type node to string type, assert should also reflect this.